### PR TITLE
SpriteKit overlay: add a better variant of the SKWarpGeometryGrid() initializer overlay

### PR DESCRIPTION
**Pull request by gribozavr**
#### Resolved bug number: ([SR-](https://bugs.swift.org/browse/SR-))

---

Before merging this pull request to apple/swift repository:
- [ ] Test pull request on Swift continuous integration.

<details>
  <summary>

Triggering Swift CI</summary>

The swift-ci is triggered by writing a comment on this PR addressed to the GitHub user @swift-ci. Different tests will run depending on the specific comment that you use. The currently available comments are:

**Smoke Testing**

| Platform | Comment |
| --- | --- |
| All supported platforms | @swift-ci Please smoke test |
| All supported platforms | @swift-ci Please smoke test and merge |
| OS X platform | @swift-ci Please smoke test OS X platform |
| Linux platform | @swift-ci Please smoke test Linux platform |

A smoke test on macOS does the following:
1. Builds the compiler incrementally.
2. Builds the standard library only for macOS. Simulator standard libraries and
   device standard libraries are not built.
3. lldb is not built.
4. The test and validation-test targets are run only for macOS. The optimized
   version of these tests are not run.

A smoke test on Linux does the following:
1. Builds the compiler incrementally.
2. Builds the standard library incrementally.
3. lldb is built incrementally.
4. The swift test and validation-test targets are run. The optimized version of these
   tests are not run.
5. lldb is tested.

**Validation Testing**

| Platform | Comment |
| --- | --- |
| All supported platforms | @swift-ci Please test |
| All supported platforms | @swift-ci Please test and merge |
| OS X platform | @swift-ci Please test OS X platform |
| OS X platform | @swift-ci Please benchmark |
| Linux platform | @swift-ci Please test Linux platform |

**Lint Testing**

| Language | Comment |
| --- | --- |
| Python | @swift-ci Please Python lint |

Note: Only members of the Apple organization can trigger swift-ci.
</details>

### CODE REVIEW

**Comment by gribozavr** [comment]
@swift-ci Please test

File: stdlib/public/SDK/SpriteKit/SpriteKit.swift [hunk 1]
```diff
@@ -22,13 +34,51 @@ public typealias SKColor = UIColor
...
+  ///
+  ///     [0]---[1]---[2]
+  ///      |     |     |
+  ///     [3]---[4]---[5]
+  ///      |     |     |
+  ///     [6]---[7]---[8]
+  ///
+  /// - Parameter columns: the number of columns to initialize the SKWarpGeometryGrid with
+  /// - Parameter rows: the number of rows to initialize the SKWarpGeometryGrid with
+  /// - Paremeter sourcePositions: the source positions for the SKWarpGeometryGrid to warp from
```
Comment (xwu) [inline stdlib/public/SDK/SpriteKit/SpriteKit.swift]: Typo: `Parameter`

**Comment by gribozavr** [comment]
@swift-ci Please smoke test OS X platform

**Comment by gribozavr** [comment]
Linux failed for unrelated reasons, retrying macOS.

File: stdlib/public/SDK/SpriteKit/SpriteKit.swift (hunk 1 above)
Comment (gribozavr) [inline stdlib/public/SDK/SpriteKit/SpriteKit.swift]: Thanks, fixed!

**Comment by gribozavr** [comment]
@swift-ci Please test OS X platform

**Pull request merged**